<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterDialog</class>
 <widget class="QDialog" name="FilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filter database</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ico.qrc">
    <normaloff>:/UI_ICO/ICO/ICO.ico</normaloff>:/UI_ICO/ICO/ICO.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="verticalSpacing">
    <number>20</number>
   </property>
   <item row="4" column="1" colspan="2">
    <widget class="QComboBox" name="logic_comboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>A small container</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>A large container</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Two small containers</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Empty set truck</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Heavy set truck</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QRadioButton" name="correct_radioButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>correct</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="logic_checkBox">
     <property name="text">
      <string>Logic</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="check_checkBox">
     <property name="text">
      <string>Check</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QSpinBox" name="channel_spinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="dateTime_groupBox">
     <property name="title">
      <string>DateTime</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QDateTimeEdit" name="start_dateTimeEdit">
        <property name="displayFormat">
         <string>yyyy/MM/dd HH:mm:ss</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QDateTimeEdit" name="end_dateTimeEdit">
        <property name="accelerated">
         <bool>false</bool>
        </property>
        <property name="showGroupSeparator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="displayFormat">
         <string>yyyy/MM/dd HH:mm:ss</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QRadioButton" name="error_radioButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>error</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="container_groupBox">
     <property name="title">
      <string>Container</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QLineEdit" name="front_lineEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="after_lineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="channel_checkBox">
     <property name="text">
      <string>Channel</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="plate_checkBox">
     <property name="text">
      <string>Plate</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="plate_lineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>start_dateTimeEdit</tabstop>
  <tabstop>end_dateTimeEdit</tabstop>
  <tabstop>front_lineEdit</tabstop>
  <tabstop>after_lineEdit</tabstop>
  <tabstop>plate_lineEdit</tabstop>
  <tabstop>channel_spinBox</tabstop>
  <tabstop>logic_comboBox</tabstop>
  <tabstop>correct_radioButton</tabstop>
  <tabstop>error_radioButton</tabstop>
  <tabstop>check_checkBox</tabstop>
  <tabstop>plate_checkBox</tabstop>
  <tabstop>dateTime_groupBox</tabstop>
  <tabstop>channel_checkBox</tabstop>
  <tabstop>container_groupBox</tabstop>
  <tabstop>logic_checkBox</tabstop>
 </tabstops>
 <resources>
  <include location="../ico.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FilterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FilterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plate_checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>plate_lineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>channel_checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>channel_spinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>logic_checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>logic_comboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>correct_radioButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>error_radioButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
